pkgs/misc/vim-plugins/overrides.nix

```diff
@@ -147,6 +147,15 @@ self: super: {
    '';
   });
 
+  deoplete-tabnine = super.deoplete-tabnine.overrideAttrs(old: {
+    buildInputs = [ curl ];
```

Comment:
For cross compilation correctness sake, this should be `nativeBuildInputs = [ curl ]`, because `curl` is only used during the build of the package. `buildInputs` is for things that will end up being used by the architecture running the package.

Reply:
@Infinisil Thanks for the tip, didn't know about `nativeBuildInputs`. Just pushed the change.